### PR TITLE
Global Users page bug fixes

### PR DESCRIPTION
**Pull request by ischolten**
Connect #2613

### CODE REVIEW

File: ui/src/admin/components/chronograf/AllUsersTableRowNew.js [hunk 1]
```diff
@@ -36,15 +37,13 @@ class AllUsersTableRowNew extends Component {
...
     const {onBlur, onCreateUser} = this.props
     const {name, provider, scheme, roles, superAdmin} = this.state
-
     const newUser = {
       name,
       provider,
       scheme,
       superAdmin,
-      roles: roles[0].id === null ? [] : roles,
+      roles: roles[0].organization === undefined ? [] : roles,
```
Comment (ebb-tide) [inline ui/src/admin/components/chronograf/AllUsersTableRowNew.js]: consider comparing roles to nullrole?

File: ui/src/admin/components/chronograf/AllUsersTableRowNew.js [hunk 2]
```diff
@@ -55,12 +54,12 @@ class AllUsersTableRowNew extends Component {
 
   handleSelectOrganization = newOrganization => {
     const newRoles = [
-      newOrganization.id === null
+      newOrganization.id === undefined
```
Comment (ebb-tide) [inline ui/src/admin/components/chronograf/AllUsersTableRowNew.js]: consider comparing newOrganization to null organization?

File: ui/src/admin/components/chronograf/AllUsersTableRowNew.js (hunk 1 above)
Comment (ebb-tide) [reply ui/src/admin/components/chronograf/AllUsersTableRowNew.js]: consider sending `roles: [role]`  and dealing with a `role` var in the rest of this file.

Comment (ischolten) [reply ui/src/admin/components/chronograf/AllUsersTableRowNew.js]: For comparing to null roles, it was returning false. Despite the content of the role being the same, the comparison was returning false.

File: ui/src/admin/components/chronograf/AllUsersTableRowNew.js (hunk 2 above)
Comment (ischolten) [reply ui/src/admin/components/chronograf/AllUsersTableRowNew.js]: See above

**Pull request merged**